packages/framework/src/Model/Mail/MailerSettingProvider.php

```diff
@@ -24,28 +19,17 @@ class MailerSettingProvider
     protected bool $deliveryDisabled;
 
     /**
-     * @param string $mailerWhitelist
      * @param string $mailerMasterEmailAddress
      * @param string $mailerDsn
      */
     public function __construct(
-        string $mailerWhitelist,
         string $mailerMasterEmailAddress,
         string $mailerDsn
     ) {
-        $this->mailerWhitelistExpressions = $mailerWhitelist !== '' ? explode(',', $mailerWhitelist) : [];
         $this->mailerMasterEmailAddress = $mailerMasterEmailAddress !== '' ? $mailerMasterEmailAddress : null;
         $this->deliveryDisabled = $mailerDsn === Mailer::DISABLED_MAILER_DSN;
     }
 
-    /**
-     * @return string[]
-     */
-    public function getMailerWhitelistExpressions(): array
```

Comment:
Because of https://github.com/shopsys/shopsys/pull/2592#discussion_r1166260182 this method stay intact, but other one `getMailerMasterEmailAddress` was deprecated in backward compatible way.